auth-service/src/main/java/com/petclinic/auth/User/UserServiceImpl.java

```diff
@@ -35,15 +36,10 @@ public User getUserById(long id) {
     @Override
     public User createUser(@Valid UserIDLessDTO userIDLessDTO) {
 
-        try {
-            log.info("Saving user with email {}", userIDLessDTO.getEmail());
-            User user = userMapper.idLessDTOToModel(userIDLessDTO);
-            return userRepo.save(user);
-        }
-        catch (DataIntegrityViolationException e){
-            throw new DataIntegrityViolationException("Duplicate email for userEmail " + userIDLessDTO.getEmail());
-        }
-
```

Comment:
I believe this was added by Jeremy for his email validation/verification, I'm wondering if maybe this should have stayed. Next class I'll bring this up to confirm.

Reply:
The issue with this is that it assumes that ALL data integrity violations are due to duplicate emails.
This is untrue as a data integrity exception can occur when referencing a non-existent field or using an invalid value (in terms of type) and a myriad of other things.